pkgs/top-level/release-alternatives.nix

```diff
@@ -0,0 +1,91 @@
+{ pkgsFun ? import ../..
```

Comment:
What's the intention of this file? Is it only  for testing purposes?

Reply:
I suppose this was used in conjunction with Hydra to test this PR. It's still worth keeping although its not release `*alternatives` but blas only.

Reply:
Yes - mainly to test that everything works. It certainly doesn't have to be included, but may help if we ever want to make a jobset to test this stuff.